src/models/Event.ts

```diff
@@ -422,12 +455,12 @@ export default class Event implements ISerializable {
     this._alliances = value;
   }
 
-  fixDate(date: any): any {
```

Comment:
@Techno11 Could you explain why this is necessary please?

Reply:
> 
> 
> @Techno11 Could you explain why this is necessary please?

Agreed, it just makes serializing it back to json difficult because there's no way of knowing if it should have the Z.

Reply:
I seem to remember there being a problem with the SQL because of the Z and when I took it out there was no problem

Reply:
That seems really weird, since the Z is part of the ISO-8601 that SQL follows...